src/App.css

```diff
@@ -0,0 +1,78 @@
+body {
```

Comment:
It's important that your design resembles as much as possible to Nelson's design 🧐 Please be sure to make the necessary changes in your CSS to solve this issue! 

Your design 
![image](https://user-images.githubusercontent.com/15807343/164538365-0d96d50c-126b-4f93-8397-e6a03e762234.png)

Nelson's design 
![Captura](https://user-images.githubusercontent.com/15807343/164538490-a7b4cec3-3b59-48d7-abae-59b842c4b0ce.PNG)

- You could try removing the white borders and give the cells in the table the distinction in color you see in the following image 🤓

![image](https://user-images.githubusercontent.com/15807343/164538751-d64ff83e-66af-4646-8be1-0fc89a7ec1cb.png)

Reply:
Done.